templates/web/base/my/my.html

```diff
@@ -91,10 +91,23 @@ <h1>[% loc('Your account') %]</h1>
         <label class="btn" for="alert_notify_none">[% loc('None') %]</label>
     </p>
     </fieldset>
-    <p>
-        <input class="btn btn--block" type="submit" value="[% loc('Update') %]">
-    </p>
 </li>
+[% current_pref = c.user.receive_questionnaires %]
+<li>
+    <fieldset>
+      <legend>[% loc('Receive questionnaires') %]:</legend>
+      <p class="segmented-control segmented-control--radio">
+        <input type="radio" name="receive_questionnaires" id="receive_questionnaires_yes" value="1"[% ' checked' IF current_pref %]>
+        <label class="btn" for="receive_questionnaires_yes">[% loc('Yes') %]</label>
+
+        <input type="radio" name="receive_questionnaires" id="receive_questionnaires_no" value="0"[% ' checked' IF !current_pref %]>
+        <label class="btn" for="receive_questionnaires_no">[% loc('No') %]</label>
+      </p>
+    </fieldset>
+</li>
+<p>
```

Comment:
Any reason this has moved outside the final `li`?

Reply:
I moved it because being inside the final `li` seemed like a really odd pattern to me (why would the submit button for the entire list be wrapped by one of the list items?) But it works either way so I can undo this bit.

Reply:
Thanks; the reason was/is because then the list button displays more as 'part of' the options being updated, whereas outside the list it spans the whole column and feels less joined to the options it is associated with (probably could be fixed with some extra styling but didn't seem worth it).